### PR TITLE
[CNVS Upgrade] Migrating components: Code Blocks

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

File: src/styles/components/code/variables.less [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Code Block Colors
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+@code-block-pretty-print-border-color:                                          color-lighten(@neutral, 10);
```
Comment (ashenden) [inline src/styles/components/code/variables.less]: Is the hyphen between `pretty` and `print` intentional here?

**Comment by ashenden** [comment]
Generally speaking, we can probably get rid of this component, as the new code element in CNVS is heavily improved.  Worth considering if we can strip this one of the roster.

Overall looks good though.

File: src/styles/components/code/variables.less (hunk 1 above)
Comment (jfurrow) [inline src/styles/components/code/variables.less]: Nope, good catch! Fixed.

**Comment by jfurrow** [comment]
Awesome!

**Pull request merged**